modules/apps/frontend-js/frontend-js-test/src/testFunctional/tests/SiteLevelWalkthrough.testcase

```diff
@@ -0,0 +1,62 @@
+@component-name = "portal-frontend-infrastructure"
+definition {
+
+	property osgi.modules.includes = "frontend-js-walkthrough-sample-web";
+	property portal.release = "true";
+	property portal.upstream = "true";
+	property testray.component.names = "Walkthrough";
+	property testray.main.component.name = "User Interface";
+
+	setUp {
+		TestCase.setUpPortalInstance();
+
+		User.firstLoginPG();
+	}
+
+	tearDown {
+		var testPortalInstance = PropsUtil.get("test.portal.instance");
+
+		if ("${testPortalInstance}" == "true") {
+			PortalInstances.tearDownCP();
+		}
+		else {
+			JSONGroup.deleteGroupByName(groupName = "Site Name");
+		}
+	}
+
+	@description = "LPS-159823. When closeable is set to true, then a close button is present in popover mode"
+	@priority = "4"
+	test CanEnableCloseable {
```

Comment:
Seems like this can be covered by a unit test. @mateusralv can you check with @diegonvs on this?

Reply:
Hey @john-co, I talked with @mateusralv on Slack and it seems fine to be covered by unit tests. I'll do it once https://github.com/brianchandotcom/liferay-portal/pull/123114 arrives on master, maybe tomorrow? 😅